src/detections/rule/matchers.rs

```diff
@@ -1976,6 +1976,25 @@ mod tests {
         check_select(rule_str, record_json_str, false);
     }
 
+    #[test]
+    fn test_wildcard_converted_starts_with_shorter_val_notdetect() {
```

Comment:
How about adding a test for the following cases?

In the case of the following test case, it would be correct to be true.

```
 enabled: true
        detection:
            selection: true
                Computer: A-HOST*
        details: 'command=%CommandLine%'
```

Reply:
Thank you for your review as well :)

The test below was intended to cover the true state, but what do you think? (Sorry if I misunderstood your intentions!)
https://github.com/Yamato-Security/hayabusa/blob/67f322f9991e6b590e8e05ead249977569156fc3/src/detections/rule/matchers.rs#L1942

Reply:
I'm sorry, I understand!  Added the following tests where the asterisk matches 0 characters :)
https://github.com/Yamato-Security/hayabusa/pull/890/commits/e0efd4b6223f30beff1d850ea62552441aa7a272